README.md

```diff
@@ -1 +0,0 @@
-# RecoMTD-TimingIDTools
```

Comment:
please do not remove the README and instead please add some text to describe the files currently kept in this repository

Reply:
Done